url.bs

```diff
@@ -88,6 +88,334 @@ valid input. User agents, especially conformance checkers, are encouraged to rep
  unclear to other developers.
 </div>
 
+<table>
+ <thead>
+  <tr>
+   <th>Error type
+   <th>Error description
+   <th>Failure
+ <tbody>
+  <tr>
+   <th colspan=3 scope=rowgroup><a href=#idna>IDNA</a>
+  <tr>
+   <td><dfn id=validation-error-domain-to-ascii>domain-to-ASCII</dfn>
+   <td>
+    <p>The result of <a abstract-op lt=ToASCII>Unicode toASCII</a> records an error while processing
+    the input domain.
+    <p class=note>[[!UTS46]] conformance does not require the reporting of precise errors, only that
+    an error has occurred. If the [[!UTS46]] implementation reports precise error codes, user agents
+    are encouraged pass those codes along.
+   <td>✅
```

Comment:
Although it may have been my suggestion, I think the green checkmarks / red Xs are confusing, because they feel too much like "success" / "failure".

Maybe "Yes" / "&middot;" similar to https://html.spec.whatwg.org/#linkTypes would work better.